### PR TITLE
[New Rule] Kerberos Traffic from Unusual Process

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/447

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/credential_access_kerberoasting_unusual_process.toml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+creation_date = "2020/11/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies network connections to Kerberos standard port and from an unusual process. On Windows only process that
+normally performs kerberos traffic from a Domain joined host is lsass.exe.
```
Comment (bm11100) [inline rules/windows/credential_access_kerberoasting_unusual_process.toml:11]:
```suggestion
Identifies network connections to the standard Kerberos port and from an unusual process. On Windows, the only process that
normally performs Kerberos traffic from a domain joined host is lsass.exe.
```

File: rules/windows/credential_access_kerberoasting_unusual_process.toml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+risk_score = 43
+rule_id = "897dc6b5-b39f-432a-8d75-d3730d50c782"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+type = "eql"
+
+query = '''
+network where event.type == "start" and network.direction == "outgoing" and
+ destination.port == 88 and source.port >= 49152 and
+ process.executable != "C:\\Windows\\System32\\lsass.exe" and destination.address!="127.0.0.1" and destination.address!="::1" and
```
Comment (bm11100) [inline rules/windows/credential_access_kerberoasting_unusual_process.toml:32]:
```suggestion
 process.executable != "C:\\Windows\\System32\\lsass.exe" and destination.address !="127.0.0.1" and destination.address !="::1" and
```
this isn't a big deal, just for consistency purposes, we should either use a space in all of them, or no space in all of them

File: rules/windows/credential_access_kerberoasting_unusual_process.toml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2020/11/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies network connections to the standard Kerberos port and from an unusual process. On Windows, the only process that
```
Comment (bm11100) [inline rules/windows/credential_access_kerberoasting_unusual_process.toml:10]:
```suggestion
Identifies network connections to the standard Kerberos port from an unusual process. On Windows, the only process that
```

File: rules/windows/credential_access_kerberoasting_unusual_process.toml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+normally performs Kerberos traffic from a domain joined host is lsass.exe.
+"""
+false_positives = [
+    """
+    HTTP Traffic on Non Standard port 88, verify that the destination IP address is not related to a Domain Controller.
```
Comment (bm11100) [inline rules/windows/credential_access_kerberoasting_unusual_process.toml:15]:
```suggestion
    HTTP traffic on a non standard port. Verify that the destination IP address is not related to a Domain Controller.
```

**Review by bm11100 — APPROVED** [review]
minor suggestions but otherwise looks good.

**Review by peasead — APPROVED** [review]
LGTM

**Pull request merged**